designsafe/static/scripts/data-depot/components/projects/pipeline-categories/pipeline-categories-field-recon.component.html

```diff
@@ -73,7 +73,7 @@ <h3 style="text-align: center; color: orange;">
     </div>
     <!-- Categories Start -->
     <!-- Reports -->
-    <div ng-repeat="report in $ctrl.browser.project.report_set | orderBy: 'value.title'"
+    <div ng-repeat="report in $ctrl.project.report_set | orderBy: 'value.title'"
```

Comment:
Does all of field-recon replace `browser.project` with `project`?

Reply:
Project metadata for all project types is assigned to `this.project` when inside the pipeline. However, I noticed that the DataBrowserService was still being imported and assigned in the controller. It's not being used here, so I should just take it out.

Reply:
There are other areas where `browser.project` is being used, should those be using `project` as well?